cf_units/tests/test_unit.py

```diff
@@ -373,8 +373,13 @@ def test_multiply_not_numeric(self):
     def test_multiply_with_unknown_unit(self):
         u = Unit('unknown')
         v = Unit('meters')
+        self.assertIsNotNone
```

Comment:
I don't think you meant this... it's not calling the method.

Reply:
I think this accidently got left in when I was writing the tests. Has been removed.